documentation/examples/rbac-setup.yml

```diff
@@ -0,0 +1,34 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: prometheus
+rules:
+- apiGroups: [""]
+  resources:
+  - nodes
+  - nodes/proxy
+  - services
+  - endpoints
+  - pods
+  verbs: ["get", "list", "watch"]
+- nonResourceURLs: ["/metrics"]
+  verbs: ["get"]
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: prometheus
+  namespace: default
```

Comment:
I can leave the namespace out of the _ServiceAccount_ spec, but it's required in the _ClusterRoleBinding_ spec, so this makes it more obvious. In the _PodSpec_ of the _Prometheus_ deployment manifest you set the service account as follows:

```yaml
    spec:
      serviceAccount: prometheus
```

Which then leads to the token mounted at `/var/run/secrets/kubernetes.io/serviceaccount/token` to be a token for that service account will all the appropriate roles set by the _ClusterRoleBinding_.

Perhaps this requires more documentation on how to set up _Prometheus_ on _Kubernetes_ in general, but perhaps that's better done in a separate PR?